okhttp-testing-support/src/main/java/okhttp3/OkHttpClientTestRule.kt

```diff
@@ -119,6 +133,19 @@ class OkHttpClientTestRule : TestRule {
     }
   }
 
+  /** Returns true if this queue became idle before the timeout elapsed. */
+  private fun TaskQueue.awaitIdle(timeout: Long, timeUnit: TimeUnit): Boolean {
+    val latch = CountDownLatch(1)
+    schedule(object : Task("awaitIdle") {
```

Comment:
Should we try to put some unique prefix on task names?  So an app with >1 okhttp client, threads can be disambiguated?

Reply:
Good idea. But unfortunately from this layer they’re all pretty symmetric.

We’d need to either name the OkHttpClient, or name its component parts, or something. Any recommendations on what the names should be?